static/sass/styles.scss

```diff
@@ -0,0 +1,8726 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: "Ubuntu";
```

Comment:
What are all these styles for if you are using Vanilla?

Reply:
Not using it anymore :) Removed the dependency